python/CHANGELOG.rst

```diff
@@ -2,6 +2,12 @@
 [0.6.1] - 2024-XX-XX
 --------------------
 
+**Bugfixes**
+
+- Fix to ``Tree.draw_svg()`` with the (undocumented) experimental
+  ``pack_untracked_polytomies=True`` option. (:user:`hyanwong`,
```

Comment:
Probably doesn't need to go in changelog as it's undocumented (we're sort of documenting it otherwise)

Reply:
Fine, I wasn't sure about that one either. Now removed.